cpp/tests/experimental/louvain_primitives.cu

```diff
@@ -0,0 +1,179 @@
+#include <experimental/graph.hpp>
+
+#include <utilities/shuffle_comm.cuh>
+
+#include <rmm/device_uvector.hpp>
+
+// TODO:
+//    The following 3 include files are not referenced in my file,
+//    however if they are not included prior to the two pattern
+//    includes below then the compilation fails.  I suspect these
+//    should be included somehow in the patterns includes below
+//
+#include <matrix_partition_device.cuh>
+#include <patterns/edge_op_utils.cuh>
+#include <utilities/host_scalar_comm.cuh>
+// END EXTRA INCLUDES
```

Comment:
You can delete this after pull updates from https://github.com/rapidsai/cugraph/pull/1399

Reply:
Thanks, I deleted these and it compiled correctly